ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -176,23 +176,18 @@ conformance_test(
         "--use-self-service-error-codes",
     ],
     test_tool_args = [
-        "--additional=MultiPartySubmissionIT",
-        "--additional=ContractIdIT:RejectV0,ContractIdIT:AcceptSuffixedV1,ContractIdIT:AcceptNonSuffixedV1",
-        "--additional=ParticipantPruningIT",
-        "--additional=AppendOnlyCommandDeduplicationParallelIT",
-        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandService",
-        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandSubmissionService",
-        "--additional=KVCommandDeduplicationIT",
-        "--exclude=CommandDeduplicationIT",  # It's a KV ledger so it needs the KV variant
-        # Disable tests targeting only multi-participant setups
-        "--exclude=ParticipantPruningIT:PRImmediateAndRetroactiveDivulgence",
-        # TODO error codes: Remove below exclusions when the pertaining self-service error codes are implemented and can be asserted
-        "--exclude=DeeplyNestedValueIT",
-        "--exclude=TransactionServiceValidationIT:TXRejectBeginAfterEnd",
-        "--exclude=ConfigManagementServiceIT:CMSetAndGetTimeModel",
-        "--exclude=ParticipantPruningIT",
-        # TODO error codes: Check flakiness
-        "--exclude=KVCommandDeduplicationIT",
+        #        "--additional=MultiPartySubmissionIT",
+        #        "--additional=ContractIdIT:RejectV0,ContractIdIT:AcceptSuffixedV1,ContractIdIT:AcceptNonSuffixedV1",
+        #        "--additional=ParticipantPruningIT",
+        #        "--additional=AppendOnlyCommandDeduplicationParallelIT",
+        #        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandService",
+        #        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandSubmissionService",
+        #        "--additional=KVCommandDeduplicationIT",
+        #        "--exclude=CommandDeduplicationIT",  # It's a KV ledger so it needs the KV variant
+        #        # Disable tests targeting only multi-participant setups
+        #        "--exclude=ParticipantPruningIT:PRImmediateAndRetroactiveDivulgence",
+        #        # TODO error codes: Check flakiness
+        "--include=KVCommandDeduplicationIT",
```

Comment:
This is the only suite that has failing tests. Will remove this line and un-comment previous lines once these tests are fixed.

Reply:
For me they also fail locally without the v2 error codes and on `main` too. Could it be indeed a flake, perhaps they have more chances of succeeding on CI (and not as the first suite, so that the JVM is already warmed up)? They are inherently racy / time-dependent.

Reply:
Ok, will adapt and open for review. Thanks @fabiotudone-da

Reply:
Validated that they are reproducible on non self service error codes enabled suites.